head/firmware/motor_hardware.c

```diff
@@ -189,6 +194,103 @@ void MX_GPIO_Init(void) {
     GPIO_InitStruct.Pull = GPIO_NOPULL;
     GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_LOW;
     HAL_GPIO_Init(GPIOA, &GPIO_InitStruct);
+
+    /*Configure GPIO pin : B1_Pin */
```

Comment:
What are we doing with this pin? We don't seem to be using it anywhere else

Reply:
removed, not being used.